modules/cli/src/main/scala/coursier/cli/install/Install.scala

```diff
@@ -44,26 +44,7 @@ object Install extends CaseApp[InstallOptions] {
       .withVerbosity(params.output.verbosity)
       .withNativeImageJavaHome(Some(graalvmHome))
 
-    if (params.installChannels.nonEmpty) {
```

Comment:
Let's keep that just-in-case, and print a deprecation warning if it's used.

Reply:
any place which I can refer, to understand the convention for marking a command deprecated

Reply:
Thank you. I have added the same and pushed.